src/ui/layout/search/ObjectSearchResult.vue

```diff
@@ -1,23 +1,28 @@
 <template>
 <div
     class="c-gsearch-result c-gsearch-result--object"
+    role="presentation"
```

Comment:
this nulls out the aria tree so that the accessibility tree can start clean from this point. This is is what the Google Search team is using on the search results tree, for example